diff -ur a/dark.gam.dml b/dark.gam.dml
--- a/dark.gam.dml	2018-04-08 09:14:53.638356400 -0500
+++ b/dark.gam.dml	2021-02-02 02:00:50.000000000 -0600
@@ -1,5 +1,11 @@
 DML1
 
+//fingerprinting to load for OMs
+FINGERPRINT
+{
+	NAME -6862 "SmallCandleFlame"
+}
+
 #script "nvscript"
 
 //lets set up the flame effects, we need small and large flames
@@ -28,7 +34,7 @@
 	"Min time" 0.6
 	"Max time" 0.7
 }
-//adding scripts for hanging candle activation - this will also add extralight as needed so the candles won't look too dark. receptrons are set up in the gamesys.
+//adding scripts for hanging candle activation - this will also add extralight as needed so the candles won't look too dark.
 +ObjProp -470 "ExtraLight"
 {
 	"Amount (-1..1)" 0.5
@@ -73,6 +79,14 @@
 	"Model 0" chdlr
 	"Model 5" chdlr
 }
++ObjProp -5072 "Scripts"
+{
+	"Script 0" ""
+	"Script 1" ""
+	"Script 2" ""
+	"Script 3" ""
+	"Don't Inherit" true
+}
 //adding scripts on the normal candles - this will also add extralight as needed so the candles won't look too dark. NVRelayTrap4 distributes waterstim upon sim to make all candles frobinert, so we can make those few that are interactive easily frobable via mission dml
 +ObjProp -181 "Scripts"
 {
@@ -138,126 +152,126 @@
 //set up receptrons on HangingLamp to make the extralight setup work
 +Receptron -470 "FireStim"
 {
-  Min "0"
-  Max "None"
+	Min "0"
+	Max "None"
 
-  Target "Me"
-  Agent "-470"
+	Target "Me"
+	Agent "-470"
 
-  Effect "add_prop"
-  {
-    "Prop Name" "ExtraLight"
-  }
+	Effect "add_prop"
+	{
+		"Prop Name" "ExtraLight"
+	}
 }
 +Receptron -470 "WaterStim"
 {
-  Min "0"
-  Max "None"
+	Min "0"
+	Max "None"
 
-  Target "Me"
+	Target "Me"
 
-  Effect "rem_prop"
-  {
-    "Prop Name" "ExtraLight"
-  }
+	Effect "rem_prop"
+	{
+		"Prop Name" "ExtraLight"
+	}
 }
 +Receptron -470 "KOGas"
 {
-  Min "0"
-  Max "None"
+	Min "0"
+	Max "None"
 
-  Target "Me"
+	Target "Me"
 
-  Effect "rem_prop"
-  {
-    "Prop Name" "ExtraLight"
-  }
+	Effect "rem_prop"
+	{
+		"Prop Name" "ExtraLight"
+	}
 }
 //set up receptrons on chandelier to make the extralight setup work
 +Receptron -53 "FireStim"
 {
-  Min "0"
-  Max "None"
+	Min "0"
+	Max "None"
 
-  Target "Me"
-  Agent "-53"
+	Target "Me"
+	Agent "-53"
 
-  Effect "add_prop"
-  {
-    "Prop Name" "ExtraLight"
-  }
+	Effect "add_prop"
+	{
+		"Prop Name" "ExtraLight"
+	}
 }
 +Receptron -53 "WaterStim"
 {
-  Min "0"
-  Max "None"
+	Min "0"
+	Max "None"
 
-  Target "Me"
+	Target "Me"
 
-  Effect "rem_prop"
-  {
-    "Prop Name" "ExtraLight"
-  }
+	Effect "rem_prop"
+	{
+		"Prop Name" "ExtraLight"
+	}
 }
 +Receptron -53 "KOGas"
 {
-  Min "0"
-  Max "None"
+	Min "0"
+	Max "None"
 
-  Target "Me"
+	Target "Me"
 
-  Effect "rem_prop"
-  {
-    "Prop Name" "ExtraLight"
-  }
+	Effect "rem_prop"
+	{
+		"Prop Name" "ExtraLight"
+	}
 }
 //set up receptrons on Candle1 to make the extralight setup work (fire is on the concretes because we don't want to make all the prop candles interactive)
 +Receptron -182 "WaterStim"
 {
-  Min "0"
-  Max "None"
+	Min "0"
+	Max "None"
 
-  Target "Me"
+	Target "Me"
 
-  Effect "rem_prop"
-  {
-    "Prop Name" "ExtraLight"
-  }
+	Effect "rem_prop"
+	{
+		"Prop Name" "ExtraLight"
+	}
 }
 +Receptron -182 "KOGas"
 {
-  Min "0"
-  Max "None"
+	Min "0"
+	Max "None"
 
-  Target "Me"
+	Target "Me"
 
-  Effect "rem_prop"
-  {
-    "Prop Name" "ExtraLight"
-  }
+	Effect "rem_prop"
+	{
+		"Prop Name" "ExtraLight"
+	}
 }
 //set up receptrons on Candle+Stick2 to make the extralight setup work (fire is on the concretes because we don't want to make all the prop candles interactive)
 +Receptron -185 "WaterStim"
 {
-  Min "0"
-  Max "None"
+	Min "0"
+	Max "None"
 
-  Target "Me"
+	Target "Me"
 
-  Effect "rem_prop"
-  {
-    "Prop Name" "ExtraLight"
-  }
+	Effect "rem_prop"
+	{
+		"Prop Name" "ExtraLight"
+	}
 }
 +Receptron -185 "KOGas"
 {
-  Min "0"
-  Max "None"
+	Min "0"
+	Max "None"
 
-  Target "Me"
+	Target "Me"
 
-  Effect "rem_prop"
-  {
-    "Prop Name" "ExtraLight"
-  }
+	Effect "rem_prop"
+	{
+		"Prop Name" "ExtraLight"
+	}
 }
diff -ur a/miss1.mis.dml b/miss1.mis.dml
--- a/miss1.mis.dml	2018-04-08 08:45:04.428019500 -0500
+++ b/miss1.mis.dml	2019-06-23 03:04:00.000000000 -0500
@@ -1,5 +1,10 @@
 DML1
 
+FINGERPRINT
+{
+	QVAR "T2origmis" == 1
+}
+
 //no setup for this chandelier
 +ObjProp 269 "Scripts"
 {
diff -ur a/miss10.mis.dml b/miss10.mis.dml
--- a/miss10.mis.dml	2018-04-08 15:49:25.929717800 -0500
+++ b/miss10.mis.dml	2019-06-23 03:04:00.000000000 -0500
@@ -1,5 +1,10 @@
 DML1
 
+FINGERPRINT
+{
+	QVAR "T2origmis" == 1
+}
+
 // adding extralight where necessary
 +ObjProp 352 "ExtraLight"
 {
diff -ur a/miss11.mis.dml b/miss11.mis.dml
--- a/miss11.mis.dml	2018-04-08 15:40:35.757782300 -0500
+++ b/miss11.mis.dml	2021-02-02 02:00:50.000000000 -0600
@@ -1,5 +1,12 @@
 DML1
 
+FINGERPRINT
+{
+	QVAR "T2origmis" == 1
+}
+
+#script "nvscript"
+
 //making the concretes interactive
 ObjProp 7 "AnimLight"
 {
diff -ur a/miss12.mis.dml b/miss12.mis.dml
--- a/miss12.mis.dml	2018-04-08 16:03:32.471137200 -0500
+++ b/miss12.mis.dml	2019-06-23 03:04:00.000000000 -0500
@@ -1,5 +1,10 @@
 DML1
 
+FINGERPRINT
+{
+	QVAR "T2origmis" == 1
+}
+
 // adding extralight where necessary
 +ObjProp 108 "ExtraLight"
 {
diff -ur a/miss13.mis.dml b/miss13.mis.dml
--- a/miss13.mis.dml	2018-04-08 16:03:23.402618500 -0500
+++ b/miss13.mis.dml	2019-06-23 03:04:00.000000000 -0500
@@ -1,5 +1,10 @@
 DML1
 
+FINGERPRINT
+{
+	QVAR "T2origmis" == 1
+}
+
 // adding extralight where necessary
 +ObjProp 887 "ExtraLight"
 {
diff -ur a/miss14.mis.dml b/miss14.mis.dml
--- a/miss14.mis.dml	2018-04-08 16:08:55.877635000 -0500
+++ b/miss14.mis.dml	2021-02-02 02:00:50.000000000 -0600
@@ -1,5 +1,12 @@
 DML1
 
+FINGERPRINT
+{
+	QVAR "T2origmis" == 1
+}
+
+#script "nvscript"
+
 //making the concretes interactive
 ObjProp 68 "AnimLight"
 {
diff -ur a/miss7.mis.dml b/miss7.mis.dml
--- a/miss7.mis.dml	2018-04-08 15:55:59.035202200 -0500
+++ b/miss7.mis.dml	2019-06-23 03:04:00.000000000 -0500
@@ -1,5 +1,10 @@
 DML1
 
+FINGERPRINT
+{
+	QVAR "T2origmis" == 1
+}
+
 // adding extralight where necessary
 +ObjProp 586 "ExtraLight"
 {
